src/components/form/head.vue

```diff
@@ -104,6 +133,14 @@ export default {
         { text: this.project.dataExists ? this.project.nameWithArchived : this.$t('resource.project'), path: this.projectPath() },
         { text: this.$t('resource.forms'), path: this.projectPath(), icon: 'icon-file' }
       ];
+    },
+    attachedDatasets() {
+      return this.publishedAttachments.data
+        .filter(attachment => attachment.datasetExists)
```

Comment:
idea for backend: consider adding `dataset` name to attachments API (can keep `datasetExists`)